.circleci/config.yml

```diff
@@ -110,11 +128,18 @@ workflows:
           filters:
             branches:
               only: main
-      - deploy:
+      - deploy_docker:
           requires:
             - build_and_test
           filters:
             tags:
               only: /.*/
             branches:
               only: main
+      - deploy_cloud_function:
+          context: data-eng-probe-scraper
```

Comment:
TODO: update this with the build context provided in [DSRE-684](https://mozilla-hub.atlassian.net/browse/DSRE-684)